app/models/post.rb

```diff
@@ -0,0 +1,14 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments
+  has_many :likes
+  after_save :update_user_posts_counter
+
+  def update_user_posts_counter
+    user.update(posts_counter: user.posts.count)
+  end
```

Comment:
- - Error screenshot:
![Selection_080](https://github.com/yusufyusie/blog_app/assets/107632780/8121b734-c254-45b9-b9e4-45f2343f8ba0)

- [ ] When I try to create a post from the console, I get the error as shown in the screenshot above. This error is caused by the `pdate_user_posts_counter` method above. According to the relationship you have defined, a post belongs to an `author` and not to a `user`. I'll suggest you update this method while keeping the relationship in mind. :+1: :100: :+1: